xdsl/irdl/common.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum
```

Comment:
Should we rename this to `error.py` for now?

Reply:
Why not!